filebeat/docs/configuration.asciidoc

```diff
@@ -63,27 +63,27 @@ The value that you specify here is used as the `input_type` for each event publi
 
 ===== exclude_lines
 
-A list of regular expressions to match the lines that are dropped. It drops the lines that are matching any regular
-expression from the list. By default, no lines are dropped.
+A list of regular expressions to match the lines that you want Filebeat to exclude. Filebeat drops any lines that match a regular expression in the list. By default, no lines are dropped.
+
+The following example configures Filebeat to drop any lines that start with "DBG".
 
 [source,yaml]
 -------------------------------------------------------------------------------------
 exclude_lines: ["^DBG"]
 -------------------------------------------------------------------------------------
-To exclude the lines starting with "DBG".
 
 ===== include_lines
 
-A list of regular expressions to match the lines that are exported. It exports only the lines that are matching any regular expression from the list. By default, all lines are exported.
+A list of regular expressions to match the lines that you want Filebeat to include. Filebeat exports only the lines that match a regular expression in the list. By default, all lines are exported.
+
+The following example configures Filebeat to export any lines that start with "ERR" or "WARN":
 
 [source,yaml]
 -------------------------------------------------------------------------------------
 include_lines: ["^ERR", "^WARN"]
 -------------------------------------------------------------------------------------
-To include only the lines starting with "ERR" or "WARN".
 
-Note::
-If both `include_lines` and `exclude_lines` are defined, then include_lines is called first. To export all the apache logs except the DBGs, then you can use:
+NOTE: If both `include_lines` and `exclude_lines` are defined, `include_lines` is called first. So, for example, to export all Apache log lines except the debugging messages (DBGs), you can use:
```

Comment:
@urso is for you the text unclear or the behaviour? It means the first one you described. I quite like the example. Perhaps we should use "executed" instead of "called"?